openapi.yaml

```diff
@@ -1055,11 +1056,16 @@ components:
             description: Hash of the associated certificate
             type: string
             format: bytes
+          created_at:
+            description: Date and time of the entity creation
```

Comment:
`Date and time at which the Mithril stake distribution was created` sounds better